helm/linkerd2-cni/charts/partials/templates/_noop.tpl

```diff
@@ -3,6 +3,15 @@ securityContext:
   runAsUser: 65535
 args:
 - -v
-image: gcr.io/google_containers/pause:3.2
+image: "{{.Values.image.registry}}/{{.Values.noop.image.name}}:{{.Values.noop.image.version}}"
 name: noop
+resources:
+  limits:
+    cpu: "50m"
+    memory: "10Mi"
+  requests:
+    cpu: "50m"
+    memory: "10Mi"
+securityContext:
+  runAsUser: {{ .Values.proxyInit.runAsUser | int | eq 0 | ternary 65534 .Values.proxyInit.runAsUser }}
```

Comment:
Keep in mind: This is not used for proxies injected by the proxy injector. Is there a config value we can set to tell proxy injector which image to use? Same for resources.

Reply:
Leaving aside this is not used for the CNI, you are right. I'm not finding a way to change that in `config time`. It seems that it can be changed with an annotation, but it's quite suboptimal https://linkerd.io/2.12/reference/proxy-configuration/

Reply:
forget it,  proxy's global configurations can be set y the linkerd-control-plane values files, but the noop is hardcoded on building time.